.license/style.xml

```diff
@@ -36,4 +36,14 @@
         <isMultiline>true</isMultiline>
         <padLines>false</padLines>
     </shell>
+    <python>
```

Comment:
Seems python same as existing shell 
Just remove space from <firstLine># </firstLine> and reuse <shell>

Reply:
fixed